src/rules/rule-config.d.ts

```diff
@@ -1,4 +1,5 @@
 import type { RuleLevel } from './rule-severity';
+export type { RuleLevel } from './rule-severity';
```

Comment:
This looks unusual to me
If you want to import `RuleLevel` from somewhere, you should directly import it from `rule-severity` and not from `rule-config`